workshops/views.py

```diff
@@ -474,6 +497,13 @@ def task_details(request, task_id):
     return render(request, 'workshops/task.html', context)
 
 
+@login_required
+def task_delete(request, task_id):
```

Comment:
On Wed, Apr 01, 2015 at 02:00:24PM -0700, Piotr Banaszkiewicz wrote:
> > +@login_required
> > +def task_delete(request, task_id):
> 
> Please add a docstring explaining where this view is used.

I'm -1 on that.  Docstring's should explain what a function (or
whatever) does, not who's using it.  It's better to pick distinct
names and use:

  $ git grep task_delete origin/pr/234

or similar to figure out who's using the function.  It's also good to
motivate odd stuff in the commit message (but this example seems
obvious enough to not need that).  And it would be good if we had some
user-facing workflow documentation, but we don't have anything along
those lines yet.